LambdatestEcom/UITestFixture.cs

```diff
@@ -51,9 +61,10 @@ await context.Tracing.StopAsync(new()
                 $"{TestContext.CurrentContext.Test.ClassName}.{TestContext.CurrentContext.Test.Name}.zip"
             )
             });
-
+            
             await page.CloseAsync();
             await browser.CloseAsync();
+            File.WriteAllText("../../../playwright/.auth/state.json", "{}");
```

Comment:
this is not needed, as main goal for me for example, is that your test loads state when you developing tests and need to run the same test again and again. In this case, if you have state file on file system, test does not login every time you run the same test